config/initializers/devise_token_auth.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 DeviseTokenAuth.setup do |config|
+  config.default_confirm_success_url = 'confirmed'
```

Comment:
So from the name and description, I would expect this to be a full URL, not just a path.

Reply:
Would  `config.default_confirm_success_url = 'localhost:3000' `be enough here at this point?